packages/contracts/markets/base/reservesConfigs.ts

```diff
@@ -179,6 +195,21 @@ export const strategyVeloWETHDAI: IReserveParams = {
   reserveFactor: ethers.utils.parseUnits('0.15',18).toString(),
 };
 
+export const strategyVelowstETHWETH: IReserveParams = {
+  strategy: rateStrategyUnborrowable,
+  baseLTVAsCollateral: ethers.utils.parseUnits('0.65',18).toString(), 
+  liquidationThreshold: ethers.utils.parseUnits('0.7',18).toString(),
+  liquidationBonus: ethers.utils.parseUnits('1.1',18).toString(), 
+  borrowingEnabled: false,
+  reserveDecimals: '18', 
+  aTokenImpl: eContractid.AToken,
+  assetType: 5, //5 is enum for velo
+  supplyCap: "130",
```

Comment:
why allow unlimited supplying of the other ones but not this one?

Reply:
this one is a aerodrome LP token. These need supply caps. Yearn tokens don't need supply caps. The other ones I think u mean the yearn tokens